metadata.json

```diff
@@ -53,5 +53,5 @@
   "source": "https://github.com/zend-patterns/ZendServerPuppet.git",
   "summary": "Install and manage Zend Server",
   "types": [],
-  "version": "0.1.0"
+  "version": "0.2.0"
```

Comment:
I propose tagging the current master as 0.1.0 before merging this pr, and then tagging 0.2.0 afterwards. This should be a BC update.